openBCIBoard.js

```diff
@@ -172,10 +207,9 @@ function OpenBCIFactory() {
     OpenBCIBoard.prototype.disconnect = function() {
         var timeout = 0;
         if (this.streaming) {
+            this.streamStop();
             if(this.options.verbose) console.log('stop streaming');
-            this.streaming = false;
-            this.write(k.OBCIStreamStop);
-            timeout = 10;
+            timeout = 60;
```

Comment:
If streaming timeout = 60 else timeout = 0... that's what logic is happening here. Also i can't stress enough how this is absolutely a case by case basis, if i comment next to each of these would that be ok?